<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminView</class>
 <widget class="QWidget" name="AdminView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>598</width>
    <height>354</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>管理员界面</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <widget class="QWidget" name="tabCompany">
      <attribute name="title">
       <string>快递公司管理</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tabUser">
      <attribute name="title">
       <string>用户管理</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tabPackage">
      <attribute name="title">
       <string>包裹管理</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tabCabinet">
      <attribute name="title">
       <string>快递柜管理</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tabProfile">
      <attribute name="title">
       <string>个人信息</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tabPickup">
      <attribute name="title">
       <string>取件管理</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tabFeedback">
      <attribute name="title">
       <string>反馈评价</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
